cmake/SSGCommon.cmake

```diff
@@ -65,7 +65,7 @@ macro(ssg_build_bash_remediation_functions)
 
     add_custom_command(
         OUTPUT "${CMAKE_BINARY_DIR}/bash-remediation-functions.xml"
-        COMMAND "${PYTHON_EXECUTABLE}" "${SSG_SHARED_UTILS}/generate-bash-remediation-functions.py" --input "${SSG_SHARED}/bash_remediation_functions" --output "${CMAKE_BINARY_DIR}/bash-remediation-functions.xml"
+        COMMAND ${CMAKE_COMMAND} -E env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_SHARED_UTILS}/generate-bash-remediation-functions.py" --input "${SSG_SHARED}/bash_remediation_functions" --output "${CMAKE_BINARY_DIR}/bash-remediation-functions.xml"
```

Comment:
According to [here](https://cmake.org/Bug/view.php?id=5145), build failure is because `cmake -E env` isn't available before CMake 3.1. I don't think we're willing to up the CMake minimum version at this time, so I'll replaced it with just `env`, though this might not play nicely on Windows. Do we support building on Windows?

Reply:
Yes, we should not use `env` for multi-platform support.

Reply:
We do not support buildling on Windows but we really really want to. The rest of cmake port was done with as much care to make sure it can be used on Windows.